packages/common/src/index.ts

```diff
@@ -938,7 +950,7 @@ export default class Common extends EventEmitter {
    *
    * Note: This value can update along a hardfork.
    */
-  consensusType(): string {
+  consensusType(): string | ConsensusType {
```

Comment:
my thought is that it would be good to unify these return types to just the enum, gabriel said:
> I initially tried that but it turns out that TypeScript doesn't consider `'pow'` as ConsensusType even if `ConsensusType.ProofOfWork = 'pow'`

so this can be a breaking change item for the future.